encode/src/main/scala/org/broadinstitute/monster/etl/encode/EncodeIngest.scala

```diff
@@ -66,9 +66,54 @@ object EncodeIngest {
     // Add processing steps between the read and write here.
     pipelineContext
       .jsonFile[JsonObject](parsedArgs.experimentsJson)
+      .map { jObj =>
+        jObj.filterKeys(Filters.EncodeFields.contains)
+      }
+      .map { jObj =>
+        Filters.RenamedEncodeFields.foldLeft(jObj) {
+          case (currentJObj, (oldFieldName, newFieldNames)) =>
+            currentJObj(oldFieldName).fold(currentJObj) { value =>
+              newFieldNames.foldLeft(currentJObj) { (currentJObjModify, newFieldName) =>
+                currentJObjModify
+                  .add(newFieldName, value)
+                  .remove(oldFieldName)
+              }
+            }
+        }
+      }
       .saveAsJsonFile(parsedArgs.outputDir)
 
     pipelineContext.close().waitUntilDone()
     ()
   }
 }
+
+object Filters {
```

Comment:
For Scala, standard practice is to only put `object`/`class` pairs into the same file if they have the same name. I think you could either:
1. Move these `val`s into the existing `EncodeIngest` object, or
2. Move the `Filters` object into its own file